composer.dev.json

```diff
@@ -182,12 +182,13 @@
         ],
         "post-install-cmd": [
             "DrupalComposer\\DrupalScaffold\\Plugin::scaffold",
-            "rm -f docroot/sites/default/settings.php",
             "DrupalProject\\composer\\ScriptHandler::createRequiredFiles",
             "Utilities\\composer\\DrupalSettings::create"
         ],
         "post-update-cmd": [
-            "DrupalProject\\composer\\ScriptHandler::createRequiredFiles"
+            "DrupalComposer\\DrupalScaffold\\Plugin::scaffold",
```

Comment:
`post-install-cmd` never ran as there was no lock file, so whenever it ran in container - there were no core files added.